inspire-team-pty-ltd/Traffee
# TRAF-25 Renamed quicksilver to Traffee

Renamed the project to Traffee.

JIRA issue: https://weareinspire.atlassian.net/browse/TRAF-25

## Prerequisites/How to test
1. build the project
2. Run again the `"SetupDatabase.cmd"` under `Setup` folder
3. If you are using IIS change the physical path `[Yourdirectory]\Traffee\Sources\InspireTeam.Traffee.Site`
4. Place back again the `Licence.config`(the one that you are genereted on licence.episerver.com) on that folder.
5. check that the site still works

## Checklist
- [ ] No code checked in that is commented out
- [ ] No hard coded dev values (emails, postal codes, names, etc)
- [ ] Code handles errors gracefully and the user is notified when such errors occur
